ajax/libs/nosleep/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "nosleep",
+  "filename": "NoSleep.min.js",
+  "version": "0.5.0",
+  "description": "Prevent display sleep on all Android and iOS browsers",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/adasq/nosleep.git"
+  },
+  "author": "Rich Tibbett",
+  "license": "MIT",
+  "homepage": "https://github.com/richtr/NoSleep.js",
```

Comment:
We'll use git repo as homepage by default if the custom homepage is not found, so there is no need to set homepage to its git repo.